tracer/build/_build/docker/smoke.chiseled.dockerfile

```
@@ -74,8 +74,8 @@ ENTRYPOINT ["dotnet", "AspNetCoreSmokeTest.dll"]
 # Because there _is_ no shell in the chiseled containers
 # Also, we can't use env vars/args in the entrypoint, hence the need for separate targets
 FROM installer-base as dd-dotnet-final-linux-x64
-ENTRYPOINT ["/opt/datadog/linux-x64/dd-dotnet", "run", "--set-env", "DD_PROFILING_ENABLED=1","--set-env", "DD_APPSEC_ENABLED=1","--set-env", "DD_TRACE_DEBUG=1", "--", "dotnet", "/app/AspNetCoreSmokeTest.dll"]
+ENTRYPOINT ["/opt/datadog/linux-x64/dd-dotnet", "run", "--", "dotnet", "/app/AspNetCoreSmokeTest.dll"]
```

I'm not sure why, but this seems to break, so reverting it for now, and can revisit. I don't think this one fails so much anyway

```suggestion
ENTRYPOINT ["/opt/datadog/linux-x64/dd-dotnet", "run", "--set-env", "DD_PROFILING_ENABLED=1","--set-env", "DD_APPSEC_ENABLED=1","--set-env", "DD_TRACE_DEBUG=1", "--", "dotnet", "/app/AspNetCoreSmokeTest.dll"]
```